Kitodo/src/main/webapp/WEB-INF/templates/includes/currentTasksEdit/taskBoxActivities.xhtml

```diff
@@ -122,71 +122,6 @@
                         <p:confirm header="#{msgs.confirmRelease}" message="#{msgs.reallyReleaseTask}}"
                                    icon="ui-icon-alert"/>
                     </p:commandLink>
-
```

Comment:
Where did this part go? Is it still possible to add a comment in the current task?

Reply:
Now the user can add a comment in the Nachricht tab, the second tab of edit current task.